pkg/kubelet/network/exec/exec.go

```diff
@@ -33,7 +33,7 @@ limitations under the License.
 //                formatted output indicating the pod's IPAddress(v4/v6). An empty string value or an erroneous output
 //                will mean the container runtime (docker) will be asked for the PodIP
 //                e.g. {
-//                         "apiVersion" : "v1beta1",
+//                         "apiVersion" : "v1",
```

Comment:
I wonder why you don't change the code on line 172, just changing the doc @feihujiang